cql3/Cql.g

```diff
@@ -1916,6 +1917,8 @@ K_GROUP:       G R O U P;
 
 K_LIKE:        L I K E;
 
+K_TIMEOUT:     T I M E O U T;
```

Comment:
Add to basic_unreserved_keyword?

Reply:
From the `basic_unreserved_keyword` name I assume it makes sense. Will do.

Reply:
done